impeller/typographer/backends/skia/text_render_context_skia.cc

```diff
@@ -27,11 +26,20 @@ TextRenderContextSkia::TextRenderContextSkia(std::shared_ptr<Context> context)
 TextRenderContextSkia::~TextRenderContextSkia() = default;
 
 static FontGlyphPair::Set CollectUniqueFontGlyphPairsSet(
+    GlyphAtlas::Type type,
     TextRenderContext::FrameIterator frame_iterator) {
   FontGlyphPair::Set set;
   while (auto frame = frame_iterator()) {
     for (const auto& run : frame->GetRuns()) {
       auto font = run.GetFont();
+      // switch (type) {
```

Comment:
Isn't a consistent font size desirable for an SDF based strategy?

Reply:
Oh, I saw your clarifying remark. Let's cross reference a bug here instead of leaving the code commented out.

Reply:
https://github.com/flutter/flutter/issues/112016, adding TODO